### PR TITLE
Video player improvements 

### PR DESCRIPTION
**Pull request by christianmemije**
## Summary

* Makes player's volume, rate, and cc persist..
* Upgrades to videojs 6.1. 
* Makes player more responsive.

## Notes
* Buttons increase size on mobile devices for a bigger touch target.

## Before
before

## After
after

### CODE REVIEW

File: kolibri/plugins/video_mp4_render/assets/src/views/index.vue [hunk 1]
```diff
@@ -26,6 +26,8 @@
   const LangLookup = require('./languagelookup');
   const customButtons = require('./videojs-replay-forward-btns');
   const throttle = require('lodash/throttle');
+  const Lockr = require('lockr');
```
Comment (rtibbles) [inline kolibri/plugins/video_mp4_render/assets/src/views/index.vue:29]: `lockr` relies on the localStorage API in the backend, which may not be supported by older versions of Android 4: http://caniuse.com/#feat=namevalue-storage

Comment (rtibbles) [reply kolibri/plugins/video_mp4_render/assets/src/views/index.vue:29]: Can add this polyfill for compatibility: https://www.npmjs.com/package/webstorage-polyfill

Comment (christianmemije) [reply kolibri/plugins/video_mp4_render/assets/src/views/index.vue:29]: Hmm
https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage 
http://dev-test.nemikor.com/web-storage/support-test/

Comment (rtibbles) [reply kolibri/plugins/video_mp4_render/assets/src/views/index.vue:29]: Hrm, guess caniuse is just not showing stuff before Android 4.4! No worries then!

**Comment by rtibbles** [comment]
Rerunning errored test.

**Pull request merged**